shared/slices/slice.go

```diff
@@ -1,5 +1,18 @@
 package slices
 
+// Contains determines if a uint64 item exists in a slice.
+func Contains(item uint64, slice []uint64) bool {
```

Comment:
We have one but w/o the` len(slice) == 0` check, we should  just add the check there and change the name if you like `Contains` better

Reply:
https://github.com/prysmaticlabs/prysm/blob/master/shared/slices/slice.go#L62

Reply:
+1

if len(slice) == 0 is unnecessary too